packages/itwinui-variables/mixins.scss

```diff
@@ -0,0 +1,275 @@
+  --iui-color-foreground-body-invert: rgb(var(--iui-color-foreground-body-invert-rgb));
+  --iui-color-foreground-accessory-rgb: 255, 255, 255;
+  --iui-color-foreground-accessory: rgb(var(--iui-color-foreground-accessory-rgb));
+  --iui-color-foreground-primary-rgb: 0, 138, 224;
+  --iui-color-foreground-primary: rgb(var(--iui-color-foreground-primary-rgb));
+  --iui-color-foreground-positive-rgb: 83, 162, 26;
+  --iui-color-foreground-positive: rgb(var(--iui-color-foreground-positive-rgb));
+  --iui-color-foreground-warning-rgb: 241, 141, 19;
+  --iui-color-foreground-warning: rgb(var(--iui-color-foreground-warning-rgb));
+  --iui-color-foreground-negative-rgb: 209, 10, 10;
+  --iui-color-foreground-negative: rgb(var(--iui-color-foreground-negative-rgb));
+  --iui-color-background-1-overlay-rgb: 242, 242, 242;
+  --iui-color-background-1-overlay: rgb(var(--iui-color-background-1-overlay-rgb));
+  --iui-color-background-primary-overlay-rgb: 0, 107, 173;
+  --iui-color-background-primary-overlay: rgb(var(--iui-color-background-primary-overlay-rgb));
+  --iui-color-background-positive-overlay-rgb: 61, 118, 19;
+  --iui-color-background-positive-overlay: rgb(var(--iui-color-background-positive-overlay-rgb));
+  --iui-color-foreground-primary-overlay-rgb: 0, 107, 173;
+  --iui-color-foreground-primary-overlay: rgb(var(--iui-color-foreground-primary-overlay-rgb));
+  --iui-color-foreground-positive-overlay-rgb: 61, 118, 19;
+  --iui-color-foreground-positive-overlay: rgb(var(--iui-color-foreground-positive-overlay-rgb));
+  --iui-color-foreground-warning-overlay-rgb: 198, 114, 12;
+  --iui-color-foreground-warning-overlay: rgb(var(--iui-color-foreground-warning-overlay-rgb));
+  --iui-color-foreground-negative-overlay-rgb: 161, 8, 8;
+  --iui-color-foreground-negative-overlay: rgb(var(--iui-color-foreground-negative-overlay-rgb));
+  --iui-opacity-1: 1;
+  --iui-opacity-2: 0.8;
+  --iui-opacity-3: 0.6;
+  --iui-opacity-4: 0.4;
+  --iui-opacity-5: 0.2;
+  --iui-opacity-6: 0.1;
+  --iui-text-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-2));
+  --iui-text-color-muted: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+  --iui-text-color-placeholder: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-5));
+  --iui-icons-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+  --iui-icons-color-actionable: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-2));
+  --iui-icons-color-actionable-hover: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-1));
+  --iui-icons-color-actionable-disabled: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-5));
+  --iui-icons-color-primary: var(--iui-color-foreground-primary);
+  --iui-icons-color-positive: var(--iui-color-foreground-positive);
+  --iui-icons-color-warning: var(--iui-color-foreground-warning);
+  --iui-icons-color-negative: var(--iui-color-foreground-negative);
+}
+
+@mixin dark-variables {
```

Comment:
So all variables are stored in mixins now.. We were talking also about possibility to use certain theme variables and with current implementation it would be possible to do ONLY with sass. Just a thought for future, so we are sure we would not need to change much.